allure-generator/src/main/javascript/blocks/label/styles.scss

```diff
@@ -11,5 +11,6 @@
 
   &__info {
     background: #3c58e4;
+    padding-bottom: 1px;
```

Comment:
you can't change anything but color for this class. We have base style `label` that contains all the styles, and color modifier such as `label__failed`, `label__broken`, `label__info` etc.

Reply:
you can try to add padding for all the labels but I guess that is not a good solution